src/utils/filterProducts.js

```diff
@@ -0,0 +1,32 @@
+export const filterProducts = (products, filterState) => {
+  const { sortBy, category, price, rating } = filterState;
+  let filteredProducts = [...products];
+
+  // filter by price b/w lowest and selected value
+  filteredProducts = products.filter((product) => product.price <= price);
```

Comment:
this is a bit unnecessary, you can replace this with one of the below snippets
```js
let filteredProducts;
// filter by price b/w lowest and selected value
filteredProducts = products.filter((product) => product.price <= price);
```
OR 
```js
let filteredProducts = products.filter((product) => product.price <= price);
```

Reply:
made the necessary changes.